unity-client/Assets/Scripts/MainScripts/DCL/Controllers/HUD/SettingsPanelHUD/Scripts/ControlsModule/SliderSettingsControlView.cs

```diff
@@ -9,17 +9,35 @@ public class SliderSettingsControlView : SettingsControlView
         [SerializeField] private Slider slider;
         [SerializeField] private TextMeshProUGUI indicatorLabel;
 
+        public Slider sliderControl { get => slider; }
```

Comment:
```suggestion
        public Slider sliderControl => slider;
```

Reply:
Done in the feedback PR: https://github.com/decentraland/explorer/pull/1784 Thanks!